.github/workflows/mvn_docker_image.yml

```diff
@@ -13,6 +13,10 @@ on:
             app_directory:
                 required: true
                 type: string
+            app_parent_directory:
```

Comment:
Where is this input used?

Reply:
native image builds, but not anymore -> rmeoved